docs/user/trusted-publishers/adding-a-publisher.md

```diff
@@ -43,6 +43,26 @@ each.
         your trusted workflow, such as requiring manual approval on each run
         by a trusted subset of repository maintainers.
 
+=== "GitLab CI/CD"
```

Comment:
I *think* we want to exclude this from this PR, and instead bundle it with https://github.com/pypi/warehouse/pull/15192 -- that way we can make these docs public on the same "flag" day as the other publishers being added.

Reply:
(And ditto for all the other doc pages -- I'll see if I can give you push rights to that PR and, if not, I'll re-push it from the ToB fork.)

Reply:
I created a PR for the docs, with the base branch being the PR you mentioned: https://github.com/pypi/warehouse/pull/15283